packages/sources/polygon-test/src/index.ts

```diff
@@ -0,0 +1,33 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { Adapter } from '@chainlink/external-adapter-framework/adapter'
+import { conversion, tickers } from './endpoint'
+import { customSettings } from './config'
+
+export const adapter = new Adapter({
+  defaultEndpoint: tickers.name,
+  name: 'POLYGON',
+  customSettings,
+  endpoints: [tickers, conversion],
+  rateLimiting: {
+    tiers: {
+      free: {
+        rateLimit1m: 5,
+        note: 'only mentions monthly limits',
+      },
+      starter: {
+        rateLimit1h: -1,
+      },
+      developer: {
+        rateLimit1h: -1,
+      },
+      advanced: {
+        rateLimit1h: -1,
+      },
+      enterprise: {
+        rateLimit1h: -1,
+      },
```

Comment:
Instead of allowing unlimited tiers, we decided every adapter should have a reasonable limit. So v3 doesn't support -1 as a rate limit. Instead could you add something like this for each of these tiers.
```
{
  rateLimit1s: 100,
  rateLimit1m: 6000,
  note: 'Considered unlimited tier, but setting reasonable limits',
}
```